modular_zubbers/code/modules/customization/species/proteans/_proteans_species.dm

```diff
@@ -103,8 +104,26 @@
 	)
 	add_verb(gainer, protean_verbs)
 
+/datum/species/protean/proc/organ_reject(mob/living/source, obj/item/organ)
+	SIGNAL_HANDLER
+
+	if(isnull(owner))
+		return
+	var/obj/item/organ/insert_organ = organ
+	var/list/slots = list(ORGAN_SLOT_BRAIN, ORGAN_SLOT_HEART, ORGAN_SLOT_STOMACH, ORGAN_SLOT_EYES)
```

Comment:
add the rejecting slots on the species datum itself so it can be edited in-round